crypto/src/main/java/pm/gnosis/crypto/utils/Extensions.kt

```diff
@@ -0,0 +1,24 @@
+package pm.gnosis.crypto.utils
+
+import pm.gnosis.model.Solidity
+import pm.gnosis.utils.asEthereumAddressString
+import pm.gnosis.utils.removeHexPrefix
+import pm.gnosis.utils.toHexString
+
+fun Solidity.Address.asEthereumAddressChecksumString(): String {
```

Comment:
Maybe we could merge this file with the CryptoExtensions (I don't care which one we keep, but maybe it is better to just have 1)